### PR TITLE
MYB-334: Interfaz del usuario

### PR DESCRIPTION
**Pull request by ignaciodevspark**
myb-334-user-interface-io
mybus_2

### CODE REVIEW

**Comment by lisandro** [comment]
Hi @ignaciodevspark even though now user location is displayed, map isn't centered to user's coordinates.

Note: Please test using the following coordinates (-38.006650, -57.594403) that it is not visible in default center coordinates

File: MyBus/MyBus/MyBusMapController.swift [hunk 1]
```diff
@@ -57,6 +57,9 @@ class MyBusMapController: UIViewController, MGLMapViewDelegate, UITableViewDeleg
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(MyBusMapController.offlinePackProgressDidChange(_:)), name: MGLOfflinePackProgressChangedNotification, object: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(MyBusMapController.offlinePackDidReceiveError(_:)), name: MGLOfflinePackProgressChangedNotification, object: nil)
         NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(MyBusMapController.offlinePackDidReceiveMaximumAllowedMapboxTiles(_:)), name: MGLOfflinePackProgressChangedNotification, object: nil)
+        
+        mapView.showsUserLocation = true
```
Comment (safink) [inline MyBus/MyBus/MyBusMapController.swift:61]: Although this solution effectively shows the user's location annotation in the map, unfortunately doesn't center the map within the user's location coordinates. I guess there's a suitable method that could work for you in MyBusMapView class: **centerMapWithGPSLocation()**

**Comment by lisandro** [comment]
Hi @ignaciodevspark , in the meantime we can use method that Seba mentioned in MyBusMapView, then it should use location manager.

**Comment by ignaciodevspark** [comment]
@lisandro I tried that but, since we don't have any accurate GPS data (other than the one provided by the map itself) it doesn't seem to work either. I've added this method 'mapView.showsUserLocation = true' which seems to be the best the the Mapbox object can do, until we get better data from the @safink location object.

**Comment by lisandro** [comment]
👍  I got it! @ignaciodevspark but what I mean is using the Mapbox object (_that seems to be the best now_) center on that location using [this approach](http://stackoverflow.com/a/9637848/5874006) with Mapbox's delegate method

**Comment by ignaciodevspark** [comment]
@lisandro  This last commit should fix everything regarging the centering.

**Comment by lisandro** [comment]
LGTM 👍

**Pull request merged**